syntax = "proto3";
option cc_enable_arenas = true;

package Ydb.Query.Internal;

import "ydb/public/api/protos/ydb_value.proto";

message ResultSetMeta {
    repeated Ydb.Column columns = 1;

    bool truncated = 2;
    bool enabled_runtime_results = 3;  // For backward compatability
    uint64 number_rows = 4;
    bool finished = 5;
}
